module/os/windows/zfs/zfs_vnops_windows.c

```diff
@@ -4762,6 +4774,11 @@ _Function_class_(DRIVER_DISPATCH)
 			Status = ioctl_disk_get_drive_geometry(DeviceObject,
 			    Irp, IrpSp);
 			break;
+		case ZPOOL_GET_SIZE_STATS:
```

Comment:
I think this ioctl is sent from pool driver to \\Device\\ZFSCTL which is already handled above in ioctlDispatcher() and not required to be handled here?

Reply:
Removed case from diskdispatcher